docs/paper/admin/how-to/get-to-vanilla.mdx

```diff
@@ -0,0 +1,38 @@
+---
+slug: /vanilla
+description: This page lists all changes that result in a different experience than vanilla.
+---
+
+# Hot to get a vanilla experience
+
+import React from 'react';
+import ConfigDocBlock from '@site/src/components/config/ConfigDocBlock';
+import VanillaWorldDefaults from '!!raw-loader!@site/config-specs/paper/vanilla/paper-world-defaults.yml'
+import VanillaGlobalDefaults from '!!raw-loader!@site/config-specs/paper/vanilla/paper-global.yml'
+import SpigotChanges from '!!raw-loader!@site/config-specs/paper/vanilla/spigot.yml'
+
+Due to the way how the Bukkit platform is built and because it fixes vanilla bugs, the gameplay experience between Vanilla and Paper can lead to inconsistencies.
+While some people might not notice those inconsistencies, especially tech players might have troubles getting their machines to work like they should.
+This page aims to provide a starting point for players, who want to get as close to vanilla as possible.
+This page is inspired by [earthcomputer's collection](https://gist.github.com/Earthcomputer/2296da33b8cc91dba81b48103c0e1fe3) of Vanilla breaking changes.
+
+:::warning
+
+This guide will only help you to get as close to vanilla as possible.
+Unfortunately, it currently is not possible to get a 100% Vanilla experience in Paper.
+
+:::
+
+## paper-world-defaults.yml
+<ConfigDocBlock data={VanillaWorldDefaults}/>
+
+## paper-global.yml
+<ConfigDocBlock data={VanillaGlobalDefaults}/>
+
+## spigot.yml
+<ConfigDocBlock data={SpigotChanges}/>
+
+## Further considerations
+
+* `settings.timeout-time` in `spigot.yml` mirrors the watchdog setting in the `server.properties` file. Keep in mind that the `spigot.yml` setting is in seconds while the default setting is in milliseconds.
+* `watchdog` in `paper-global.yml` controls the watchdog early warning. While this is only a warning you might want to disable it as it can get annoying.
```

Comment:
```suggestion
* `watchdog` in `paper-global.yml` controls the watchdog early warning. While this is only a warning, you might want to disable it as it can get annoying.
```